_data/simple-icons.json

```
@@ -10413,6 +10413,12 @@
             "hex": "3EA8FF",
             "source": "https://zenn.dev/mediakit"
         },
+        {
+            "title": "Zenodo",
+            "hex": "000000",
+            "source": "https://about.zenodo.org/static/img/logos/zenodo-black.svg",
```

Linking to a specific file can be a bit fragile as files tend to be moved around - hence we use the page on which the SVG is found as a source.

```suggestion
            "source": "https://about.zenodo.org",
```